pkg/cmdlets/errors/errors.go

```diff
@@ -146,16 +146,23 @@ func ReportError(err error, cmd *captain.Command, an analytics.Dispatcher) {
 
 	// Log error if this isn't a user input error
 	var action string
+	errorMsg := err.Error()
 	if !locale.IsInputError(err) {
 		multilog.Critical("Returning error:\n%s\nCreated at:\n%s", errs.Join(err, "\n").Error(), stack)
 		action = anaConst.ActCommandError
 	} else {
 		action = anaConst.ActCommandInputError
+		for err != nil {
+			if locale.IsInputErrorNonRecursive(err) {
+				errorMsg = locale.ErrorMessage(err)
+				break
+			}
+			err = errors.Unwrap(err)
+		}
```

Comment:
You forgot to update `errorMsg`.

Also, you can probably simplify this by using the first slice entry returned by `locale.UnwrapError()`.

Reply:
Do we want to update the error to a special static message if we exit the for loop without encountering an input error? `errorMsg` is set on line 149 and updated when we encounter the input error.

The first entry in `locale.UnwrapError()` is not guaranteed to be an input error, just a localized one.

Reply:
Ah I forgot about that distinction! 

Looking at it again, I think your logic is sound! Makes sense to use the non-input error if we didn't encounter one (which shouldn't happen if the input error functions work as intended).